projects/schematics/src/shared/utils/file-utils_spec.ts

```diff
@@ -864,4 +950,95 @@ describe('File utils', () => {
       expect(lines[0]).toEqual(content.indexOf(testLine));
     });
   });
+
+  describe('removeInjectImports', () => {
+    it('should remove injection token AND Inject decorator imports when there is one decorator in the constructor', () => {
+      const sourcePath = 'xxx.ts';
+      const source = ts.createSourceFile(
+        sourcePath,
+        SINGLE_DECORATOR_CONSTRUCTOR,
+        ts.ScriptTarget.Latest,
+        true
+      );
+      const paramToRemove: ClassType = {
+        className: PLATFORM,
+        staticType: ANY_TYPE,
+        injectionToken: {
+          token: PLATFORM_ID_STRING,
+          importPath: ANGULAR_CORE,
+        },
+      };
+
+      const nodes = getSourceNodes(source);
+      const constructorNode = findConstructor(nodes);
+      const changes = removeInjectImports(
+        source,
+        constructorNode,
+        paramToRemove
+      );
+      expect(changes.length).toEqual(2);
+      expect(changes[0].description).toEqual(
+        `Removed Inject, into position 10 of ${sourcePath}`
+      );
+      expect(changes[1].description).toContain(
+        `Removed PLATFORM_ID into position 30 of ${sourcePath}`
+      );
+    });
+    it('should remove ONLY injection token when there are many decorators in the constructor', () => {
+      const sourcePath = 'xxx.ts';
+      const source = ts.createSourceFile(
+        sourcePath,
+        MULTIPLE_DECORATOR_CONSTRUCTOR,
+        ts.ScriptTarget.Latest,
+        true
+      );
+      const paramToRemove: ClassType = {
+        className: PLATFORM,
+        staticType: ANY_TYPE,
+        injectionToken: {
+          token: PLATFORM_ID_STRING,
+          importPath: ANGULAR_CORE,
+        },
+      };
+
+      const nodes = getSourceNodes(source);
+      const constructorNode = findConstructor(nodes);
+      const changes = removeInjectImports(
+        source,
+        constructorNode,
+        paramToRemove
+      );
+      expect(changes.length).toEqual(1);
+      expect(changes[0].description).toContain(
+        `Removed PLATFORM_ID into position 30 of ${sourcePath}`
+      );
+    });
+  });
+
+  describe('hasDuplicateDecorator', () => {
+    it('should return false if the decorator is present one time', () => {
```

Comment:
Don't forget to update this test description after adapting the `hasDuplicateDecorator`